doc/_templates/customindex.html

```diff
@@ -38,13 +38,22 @@ <h2>Open Standard</h2>
     </div>
   </div>
   <div>
-    <h2>What is Autocrypt? What is the status?</h2>
+    <h2>What is Autocrypt?</h2>
     <p>
-      Autocrypt is a specification which mail application programmers can implement
-      to offer their users convenient end-to-end encryption for e-mail. The first specification
-      was released in December 2017 and a few mail programs already have or are heading 
-      towards Autocrypt-capable releases in spring 2018.  Please join us in implementing
-      and refining "Level 1" and other upcoming specifications. For any further questions
+      Autocrypt is not an app, but there are autocrypt-enabled
+      mail-apps. Autocrypt is a specification for convenient
```

Comment:
I'd strike the first sentence (Autocrypt is not an app ...).  It's also not a tool, not a software packages (what some commenters on heise believed) etc.  let's just directly say it's a specification.  somehow even end-users will likely guess that they can't install a specification especially with the follow up sentences.

Reply:
It is polemic, yes. But I think we can afford some polemics in the first sentence - they are cleared up directly afterwards, but improve understandability a lot.

With this sentence, people will know, what they can expect and what not. It helps to sort the complicated stuff into the right drawers.

Reply:
i didn't say it's polemic. also what autocrypt-enabled mail apps are there? The ones under "Test it" are not single-click there, are they?  Also "mail app" makes sense to you, but i have talked to people who aren ot familiar with this term because they consider mail to be "gmail" or "riseup" things (i.e. web interface in our terms).  

so I tentatively insist that just starting with "autocrypt is a specification" is best thing to do here.